Formula/truffle.rb

```diff
@@ -14,6 +14,7 @@ class Truffle < Formula
     sha256 mojave:        "371f862ca3584732e155953d816355a8ecb0db7c45170cc97c884efcdb165692"
   end
 
+  depends_on arch: :x86_64
```

Comment:
Let's add a comment here to explain this too.

Reply:
👍